src/permutations.jl

```diff
@@ -172,6 +172,7 @@ end
 
 """
     multiset_permutations(m, f, t)
+    multiset_permutations(a, t)
```

Comment:
```suggestion
    multiset_permutations(a, t)
```
It looks like a documentation typo.

The test only tests functions with two arguments, and `multiset_permutations(m, f, t)` is more of an implementation detail that should probably be renamed `_multiset_permutations`.
https://github.com/JuliaMath/Combinatorics.jl/blob/d800d9fff33181dda1443c624f18319e64e6cf49/test/permutations.jl#L58-L67